packages/plugins/rubygem-foreman_openbolt/rubygem-foreman_openbolt.spec

```diff
@@ -0,0 +1,99 @@
+# template: foreman_plugin
+%global gem_name foreman_openbolt
+%global plugin_name openbolt
+%global foreman_min_version 3.14.0
+
+Name: rubygem-%{gem_name}
+Version: 0.0.1
+Release: 1%{?foremandist}%{?dist}
+Summary: Foreman OpenBolt integration
+License: GPL-3.0-only
+URL: https://github.com/overlookinfra/foreman_openbolt
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: foreman >= %{foreman_min_version}
+BuildRequires: foreman-assets >= %{foreman_min_version}
+BuildRequires: foreman-plugin >= %{foreman_min_version}
+Requires: ruby >= 2.7
+Requires: ruby < 4
+BuildRequires: ruby >= 2.7
+BuildRequires: ruby < 4
+BuildRequires: rubygems-devel
+BuildRequires: (rubygem(foreman-tasks) >= 11.0 with rubygem(foreman-tasks) < 12 with rubygem(foreman-tasks) >= 11.0.6)
+BuildArch: noarch
+Provides: foreman-plugin-%{plugin_name} = %{version}
+# end specfile generated dependencies
+
+# start package.json devDependencies BuildRequires
+BuildRequires: (npm(@babel/core) >= 7.7.0 with npm(@babel/core) < 8.0.0)
+BuildRequires: (npm(@theforeman/builder) >= 6.0.0 with npm(@theforeman/builder) < 7.0.0)
```

Comment:
From the build logs:
```
Problem: nothing provides requested (npm(@theforeman/builder) >= 6.0.0 with npm(@theforeman/builder) < 7.0.0)
```

We updated to 14.0.0 in https://github.com/theforeman/foreman-packaging/commit/f68eb49679f3858bc4e17ef312004ef8e3599d0e so you'll need to become compatible with that. There's a decent chance it'll be a noop.

Reply:
It's extracted from https://github.com/overlookinfra/foreman_openbolt/blob/137a63433ac5761aa47ae8bce6d3568e8e6a55d6/package.json#L30